app/src/main/java/com/nibalk/tasky/navigation/NavigationRoot.kt

```diff
@@ -109,5 +121,23 @@ private fun NavGraphBuilder.agendaGraph(navController: NavHostController) {
                 )
             )
         }
+        composable<AgendaEditorScreen> { backStackEntry ->
+            val args = backStackEntry.toRoute<AgendaEditorScreen>()
+            EditorScreenRoot(
+                onBackClicked = {
+                    navController.popBackStack()
```

Comment:
For user-initiated back stack popping, I'd prefer `navigateUp`, since that is aware of your back stack and will only navigate to the previous screen (and not generally just pop one from the back stack whatever it is).
When this is done by `popBackStack`, clicking a button and there's a transition animation or so, it can happen that the user could press the button multiple times when tapped fast. That would clear more of the back stack as you'd want here